spec/model/vm_spec.rb

```diff
@@ -40,6 +40,12 @@
       expect(vm.mem_gib).to eq 120
     end
 
+    it "handles standard-60" do
```

Comment:
Not sure if we need 4 tests for the same functionality as they are coming from same formula, but it you want to keep the tests in case for possible formula split in the future you can do this instead.

```ruby
it "handles mem_gib calculation" do
  vm.family = "standard"
  [1, 8 , 15, 30].each do |cores|
    expect(vm).to receive(:cores).and_return(cores)
    expect(vm.mem_gib).to eq cores * 8
  end
end
```